### ISSUE TITLE
Support for German Google Drive? (RemoteDriveRootPath="Meine Ablage")

### ISSUE THREAD

**Issue by devurandom** [odeke-em/drive#1015]
When I run `drive ls`, all paths begin with `/Meine Ablage`. However, trying to sync any of these has no effect, even if the local directory is totally empty and the remote directory contains files. I would expect `diff` and `pull` to show differences between an empty local dir and a non-empty remote dir and pull accordingly.

My suspicion is that this because my Google Drive "is German", i.e. the folder is named "Meine Ablage" instead of "My Drive". Probably `RemoteDriveRootPath` needs to be set according to the user's drive's language.

**Comment by devurandom** [comment]
Indeed, setting `RemoteDriveRootPath` to `Meine Ablage` and recompiling `drive` will make the prefix vanish and `diff` show a list of files that only exist on the remote.

**Comment by odeke-em** [comment]
Hello there @devurandom, thank you for the report and welcome to drive!

Interesting, and my apologies, we mostly in the western world design things not thinking about the rest of the world. In deed, if you change 
https://github.com/odeke-em/drive/blob/e2932d8e5e259e86abdf902fbacce756daebf260/src/misc.go#L49
to your language then it will properly register it.

Some of the options that we can do are:
1) Crowd source some translations in different languages e.g like you've provided in German and modify 
```diff
diff --git a/src/misc.go b/src/misc.go
index 5ccdffe..e417b8e 100644
--- a/src/misc.go
+++ b/src/misc.go
@@ -231,7 +231,12 @@ func rootLike(p string) bool {
 }
 
 func remoteRootLike(p string) bool {
-	return p == RemoteDriveRootPath
+	switch p {
+	case "My Drive", "Meine Ablage":
+		return true
+	default:
+		return false
+	}
 }
```

2) Translate by locales -- but this is quite complex
3) If there was an option to ask Google Drive's remote API for "My Drive" by some symbol and get that translated irrespective of language, then that would work.

**Comment by odeke-em** [comment]
Thanks again @devurandom, I've updated that translation with https://github.com/odeke-em/drive/pull/1020. Please get the latest from master and I encourage everyone to feel free to
add more translations to cover their languages as Google Drive shows on
<img width="268" alt="screen shot 2018-06-30 at 4 23 04 pm" src="https://user-images.githubusercontent.com/4898263/42129496-130c3c92-7c83-11e8-983e-856c8389a3ea.png">

**Comment by chevillet** [comment]
@odeke-em "My Drive" is called "Mon Drive" in French. It's dumb because "Drive" is obviously not French but that's what it's called anyway!

**Comment by odeke-em** [comment]
Great, thank you @chevillet! Would you like to submit a PR like I did in #1020? :)

**Comment by chevillet** [comment]
I'm on my phone right now but I will.

**Comment by chevillet** [comment]
Better late... #1021
